docs/guides/Gruntfile.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function(grunt) {
+  //to use via grunt, first load the task
+  require("load-grunt-tasks")(grunt); // npm install --save-dev load-grunt-tasks
+
+  grunt.initConfig({
+    markdownlint: {
+      full: {
+        options: {
+          config: {
+            // configure the linting rules
+            default: false, // in the beginning disable all rules and enable them on a one-by-one basis
+            MD004: { // unordered list style
+              style: "consistent"
+            },
+            MD005: true, // inconsistent indentation for list items at the same level
+            MD006: true, // consider starting bulleted lists at the beginning of the line
+            MD007: { // unordered list indentation
+              indent: 2

```

Comment:
This shouldn't work. Afaik, markdown requires at least 4 spaces indentation for lists to be recognized at the next level

Reply:
will fix